examples/bri-3/prisma/migrations/20240201082211_add_key_type_enum_to_public_key/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - Changed the type of `type` on the `PublicKey` table. No cast exists, the column would be dropped and recreated, which cannot be done if there is data, since the column is required.
+
+*/
+-- CreateEnum
+CREATE TYPE "KeyType" AS ENUM ('ECDSA', 'EDDSA');
+
+-- AlterTable
+ALTER TABLE "PublicKey" DROP COLUMN "type",
```

Comment:
can we just merge this and the migration above into a single one?
Revert the db to the state before these 2 migrations, delete them and then trigger a single migration creation

Reply:
Done.